### PR TITLE
Add build failure metric and tweak formatting

### PR DESCRIPTION
**Pull request by boshek**
Adds:
 - a table that summarizes days since last successful build. 
 - splits plot into panels
 - small code cleanups

Rendered report attached as GitHub has trouble enabling pages on my fork.
[crossbow-nightly-report.html.zip](https://github.com/ursacomputing/crossbow/files/8872806/crossbow-nightly-report.html.zip)

@jonkeane your 👀 would be appreciated though certainly no rush

### CODE REVIEW

File: crossbow-nightly-report.Rmd [hunk 1]
```diff
@@ -87,22 +84,84 @@ nightly %>%
...
+    #  locations = cells_body(
+    #   rows = Sorter == 0
+    # )) %>% 
+    #     tab_style(
+    # style = list(
+    #   cell_fill(color = "#c5fb9f")
+    #   ),
+    #  locations = cells_body(
+    #   rows = Sorter == 1
+    # )) %>%
```
Comment (assignUser) [inline crossbow-nightly-report.Rmd:154]: 👆 https://lintr.r-lib.org/reference/commented_code_linter.html ;)

**Comment by assignUser** [comment]
This is looks great! Love the triangles for 100% jobs. The table with days-since-fail is really nice, a col with the last successful commit would useful for troubleshooting.

File: crossbow-nightly-report.Rmd (hunk 1 above)
Comment (boshek) [reply crossbow-nightly-report.Rmd]: thanks

**Comment by boshek** [comment]
changes made @assignUser. Question - what would be the easiest way to resolve the conflict here? I only ask because there are layer of CI that I'm not totally aware of here so I am wondering if something more than a rebase is required. Thanks!

**Comment by assignUser** [comment]
Nice! Just rebase and drop the automatic commits, the report will be regenerated on merge anyway :)

**Review by assignUser — APPROVED** [review]
🚀

**Pull request merged**